### ISSUE TITLE
ignore w3c validator errors by type?

### ISSUE THREAD

**Issue by jm3** [alexdunae/w3c_validators#15]
is is possible to pass an options hash to validator to filter out errors that aren't relevant to your use case?

e.g. if I don't care about missing `alt` attributes on `<img>` tags, can I ask w3c_validator to ignore errors of type `[Required attribute 'X' not specified](https://validator.w3.org/docs/errors.html#ve-127)'?

**Comment by doc75** [comment]
@jm3 indeed, at the moment the new validator (NuValidator) does return only error message, not error numbers. Therefore it is a little bit more complex to handle.
Not sure how this can be done easily.
